raft-tech/TANF-app
# 2054 Owasp scans copied to prod DAC

## Summary of Changes
_Provide a brief summary of changes_

When the nightly owasp job runs a different project slug will be passed to the DAC upload script for staging vs prod.

Pull request closes #2054 

## How to Test
_List the steps to test the PR_
Log in to staging or develop and check the scans in the admin page.
https://tdp-backend-staging.app.cloud.gov/admin/security/owaspzapscan/
https://tdp-backend-develop.app.cloud.gov/admin/security/owaspzapscan/

## Deliverables
_More details on how deliverables herein are assessed included [here](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:
- [x] Successful production owasp nightly scan reports are saved in production backend (i.e. `https://api-tanfdata.acf.hhs.gov/admin/security/owaspzapscan/`) 
- [x] Successful dev and staging nightly scan reports are saved in respective non-prod backends
- [x] Errors are raised when artifacts cannot be copied to backend environment. 

### [Deliverable 2: Tested Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

Are all areas of code introduced in this PR meaningfully tested?
Are code coverage minimums met?
 - [x] Frontend coverage: [94.29% %] (see CodeCov Report comment in PR)
 - [x] Backend coverage: [95.20% %] (see CodeCov Report comment in PR)

### [Deliverable 3: Properly Styled Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)


### [Deliverable 4: Accessible](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)


_n/a: devops_ 

### [Deliverable 5: Deployed](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)

_n/a: devops_ 

### [Deliverable 6: Documented](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)


### [Deliverable 7: Secure](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)
